### PR TITLE
add hiera 5 backends

### PR DESCRIPTION
**Pull request by maxadamo**
I have added a 3rd row to list the hiera 5 backends.

### CODE REVIEW

**Comment by Dan33l** [comment]
@maxadamo you need to rebase, your clone is some commits behind.

File: _data/tools/hiera.json [hunk 1]
```diff
@@ -7,86 +7,120 @@
...
       "url": "https://github.com/binford2k/hiera-rest",
+      "name_v5": "",
+      "url_v5": "",
       "description": "Backend that allows one to look up parameters via calls to a REST endpoint."
     },
     {
       "name": "hiera-vault",
       "url": "https://github.com/jsok/hiera-vault",
+      "name_v5": "hiera_vault",
+      "url_v5": "https://forge.puppet.com/petems/hiera_vault",
```
Comment (Dan33l) [inline _data/tools/hiera.json]: To be consistent we can point to git repository https://github.com/petems/petems-hiera_vault

File: _data/tools/hiera.json [hunk 2]
```diff
@@ -7,86 +7,120 @@
...
       "name": "hiera-etcd",
       "url": "https://github.com/garethr/hiera-etcd",
+      "name_v5": "",
+      "url_v5": "",
       "description": "Etcd distributed configuration store backend."
     },
     {
       "name": "hiera-eyaml",
       "url": "https://github.com/TomPoulton/hiera-eyaml",
+      "name_v5": "",
```
Comment (Dan33l) [inline _data/tools/hiera.json]: It should be `hiera-eyaml` with `url_v5` https://github.com/voxpupuli/hiera-eyaml

File: _data/tools/hiera.json (hunk 1 above)
Comment (maxadamo) [reply _data/tools/hiera.json]: changed accordingly

File: _data/tools/hiera.json (hunk 2 above)
Comment (maxadamo) [reply _data/tools/hiera.json]: changed accordingly

**Pull request merged**